compoments/textInput.js

```diff
@@ -66,17 +66,17 @@ class TextInput extends Component {
   render(){
     return (
       <>
-        <h1 className="govuk-heading-l">{this.title}</h1>
+        <h1 id={`${this.name}-title`}  className="govuk-heading-l">{this.title}</h1>
```

Comment:
Were the IDs added to make testing easier? Or does this improve the HTML in terms of accessibility? I've seen instances where `data-testid` was used, so I was just curious if this would have been a similar case.

Reply:
Yes, they were only added to make testing easier.

We do have usages of `data-testid` to aid in testing already.

Happy to use `data-testid` and update tests and component to use it.
Will convert newly add `id`s to `data-testid` and add `data-testid` where `id` already existed.